packages/next/build/utils.ts

```diff
@@ -126,227 +134,269 @@ export async function printTreeView(
-    const border =
-      i === 0
-        ? arr.length === 1
-          ? '─'
-          : '┌'
-        : i === arr.length - 1
-        ? '└'
-        : '├'
-
-    const pageInfo = pageInfos.get(item)
-    const ampFirst = buildManifest.ampFirstPages.includes(item)
-    const totalDuration =
-      (pageInfo?.pageDuration || 0) +
-      (pageInfo?.ssgPageDurations?.reduce((a, b) => a + (b || 0), 0) || 0)
-
-    const symbol =
-      item === '/_app' || item === '/_app.server'
-        ? ' '
-        : pageInfo?.static
-        ? '○'
-        : pageInfo?.isSsg
-        ? '●'
-        : pageInfo?.runtime === SERVER_RUNTIME.edge
-        ? 'ℇ'
-        : 'λ'
-
-    usedSymbols.add(symbol)
-
-    if (pageInfo?.initialRevalidateSeconds) usedSymbols.add('ISR')
+      }
+    })
+
+    const sharedFilesSize = stats.router[routerType]?.common.size.total
+    const sharedFiles = stats.router[routerType]?.common.files ?? []
 
     messages.push([
-      `${border} ${symbol} ${
-        pageInfo?.initialRevalidateSeconds
-          ? `${item} (ISR: ${pageInfo?.initialRevalidateSeconds} Seconds)`
-          : item
-      }${
-        totalDuration > MIN_DURATION
-          ? ` (${getPrettyDuration(totalDuration)})`
-          : ''
-      }`,
-      pageInfo
-        ? ampFirst
-          ? chalk.cyan('AMP')
-          : pageInfo.size >= 0
-          ? prettyBytes(pageInfo.size)
-          : ''
-        : '',
-      pageInfo
-        ? ampFirst
-          ? chalk.cyan('AMP')
-          : pageInfo.size >= 0
-          ? getPrettySize(pageInfo.totalSize)
-          : ''
-        : '',
+      '+ First Load JS',
+      typeof sharedFilesSize === 'number' ? getPrettySize(sharedFilesSize) : '',
+      '',
     ])
+    const sharedCssFiles: string[] = []
+    ;[
+      ...sharedFiles
+        .filter((file) => {
+          if (file.endsWith('.css')) {
+            sharedCssFiles.push(file)
+            return false
+          }
+          return true
+        })
+        .map((e) => e.replace(buildId, '<buildId>'))
+        .sort(),
+      ...sharedCssFiles.map((e) => e.replace(buildId, '<buildId>')).sort(),
+    ].forEach((fileName, index, { length }) => {
+      const innerSymbol = index === length - 1 ? '└' : '├'
+
+      const originalName = fileName.replace('<buildId>', buildId)
+      const cleanName = getCleanName(fileName)
+      const size = stats.sizes.get(originalName)
+
+      messages.push([
+        `  ${innerSymbol} ${cleanName}`,
+        typeof size === 'number' ? prettyBytes(size) : '',
+        '',
+      ])
+    })
+  }
 
-    const uniqueCssFiles =
-      buildManifest.pages[item]?.filter(
-        (file) => file.endsWith('.css') && sizeData.uniqueFiles.includes(file)
-      ) || []
-
-    if (uniqueCssFiles.length > 0) {
-      const contSymbol = i === arr.length - 1 ? ' ' : '├'
+  // If enabled, then print the tree for the app directory.
+  if (lists.app && stats.router.app) {
+    messages.push(['App', '', ''])
```

Comment:
Maybe we put this in parenthesis next to `Route` instead as it seems redundant to list `pages` and then `page` here:

<img width="499" alt="Screen Shot 2022-08-03 at 6 31 39 PM" src="https://user-images.githubusercontent.com/22380829/182730060-7d8571c4-fb66-402f-b38b-82ba0f2d7c8a.png">